components/Buttons/DeleteButton.js

```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components";
+
+export default function DeleteButton({ id, onDelete }) {
```

Comment:
This component should only have the prop onDelete. It should be handed to the component like this:
```
<DeleteButton onDelete={() => myDeleteFunction(id)>delete me</DeleteButton>}
```